test/functional/kernel/hyperplane/tests/test-kernel-hyperplane-3D.hpp

```diff
@@ -62,47 +177,14 @@ KernelHyperplane3DTestImpl(const int groups, const int idimin, const int jdimin,
   std::iota( test_array, test_array + array_length, 1 );
 
   work_res.memcpy( work_array, test_array, sizeof(DATA_TYPE) * array_length );
+  using ReducerType = typename ReducerHelper<DATA_TYPE, REDUCE_POLICY, USE_PARAM_REDUCERS::value>::type;
+  ReducerType trip_count(0);
+  ReducerType oob_count(0);
```

Comment:
Can we just always pass in a reference to a `DATA_TYPE` and make a reducer or pass in as a param in `CallKernel`?